src/tests/fixtures/request.fixture.js

```diff
@@ -1,25 +1,47 @@
 import faker from 'faker';
 import RequestService from '../../services/request.service';
 import models from '../../models';
-import { createManagers, loggedInManager2 } from './managers.fixture';
-import { cleanDb } from './users.fixture';
+import { loggedInManager1 } from './managers.fixture';
+import BcryptService from '../../services/bcrypt.service';
+import JwtService from '../../services/jwt.service';
+import { defaultPreferences } from './users.fixture';
 
 
-const { Request } = models;
+const { Request, Users, Preferences } = models;
 
-const createRequests = async () => {
-  await cleanDb();
-  const manager1 = await createManagers();
-  const pendingRequest = {
-    id: faker.random.number({ min: 1, max: 5 }),
-    requesterId: loggedInManager2.id,
-    tripId: faker.random.number(),
-    status: 'pending',
-    lineManagerId: manager1.id
-  };
+const requestor = {
+  id: faker.random.number({ min: 100, max: 150 }),
+  firstName: faker.name.firstName(),
+  lastName: faker.name.lastName(),
+  email: faker.internet.email(),
+  password: BcryptService.hashPassword(faker.internet.password()),
+  isVerified: true,
+  lineManagerId: loggedInManager1.id,
+  createdAt: new Date(),
+  updatedAt: new Date(),
+};
+
+export const requestorToken = JwtService.generateToken({
+  id: requestor.id,
+  firstName: requestor.firstName,
+  lastName: requestor.lastName,
+  email: requestor.email
+});
+
+export const pendingRequest = {
+  id: faker.random.number({ min: 1, max: 5 }),
+  requesterId: requestor.id,
+  tripId: faker.random.number({ min: 10, max: 19 }),
+  status: 'pending',
+  lineManagerId: loggedInManager1.id
+};
```

Comment:
https://github.com/andela/the_spinners-backend/pull/48/files#r368473034

Reply:
@adesege good morning, here you suggested to use spread operator instead of having multiple common objects but these objects are different. one is `requestor (a user who is making a request) ` another one is `requestorToken` and the last one is `pendingRequest`. So, their data really differ.